core/server/server.js

```diff
@@ -112,9 +111,6 @@ const publicConfigSchema = Joi.object({
   },
   redirectUrl: optionalUrl,
   rasterUrl: optionalUrl,
-  cors: {
-    allowedOrigin: Joi.array().items(optionalUrl).required(),
-  },
```

Comment:
I've completely removed the `allowedOrigin`/`ALLOWED_ORIGIN` setting here as the 'suggest' endpoint was the only thing using it. There's a couple of things to consider here.

We could just leave this here as a setting that does nothing in case we want it again in future. I can see CORS allowed origins being a thing we could use again at some point and removing the setting then adding it in again might be kind of annoying for self-hosting users?

We could consider not removing the setting and issueing some kind of deprecation warning for a bit first. The effect of just removing the setting is that if you configure using env vars the `ALLOWED_ORIGIN` will just be ignored. If you have a yaml config file with `allowedOrigin` in it, Joi will fail to validate your config on startup. I feel like we've previously discussed how we deprecate/remove settings in the past and decided that just removing them like this is OK for that reason, but I can't find it. Anyone remember?

Reply:
I'd suggest we leave it in for the moment, and then open an issue to cover removing. I feel like that allows us to move forward, ensure we have something to keep track of the removal, and gives us more time to find out what our posture was/is on these types of things (I don't remember either :smile:)

Reply:
OK cool. I've pushed a6165e7 adding that back in (plus a comment explaining)